features/admin_views_future_auction.feature

```diff
@@ -2,12 +2,10 @@ Feature: Admin views future auction
   As an admin of the Micro-purchase system
   I want to be able to unpublish future auctions
 
-  Background:
+  Scenario: Viewing the auction page
```

Comment:
same as `before` statements -- it takes the dependencies of the test outside of the test. When I read a spec, I want to know all of the required pre-steps when reading it. Scrolling up to a `background` or `before` statement isn't so bad in a case like this, but when you become overly dependent on those things you suddenly have a very complex setup and are not sure which tests require it or not. Rather than carefully deciding when things have gone "too far", I find it easier to just not use `background` or `before`